src/Microsoft.ApplicationInsights.AspNet/JavaScript/IApplicationInsightsJavaScript.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Microsoft.ApplicationInsights.AspNet.JavaScript
+{
+    public interface IJavaScriptSnippet
```

Comment:
Idea here is that somebody may implement snippet that populates additional properties like application version or request ID. Or implement a snippet with ajax support.

BTW, what do you think of moving this functionality to JavaScript NuGet and take a reference on it?